049-SQLDedicatedPoolPerf/Coach/Solution-01.md

```diff
@@ -0,0 +1,63 @@
+# Challenge 01 - Ingest Data From Blob Storage - Coach's Guide 
+
+**[Home](./README.md)** - [Next Solution >](./Solution-02.md)
+
+# Notes & Guidance
+
+In this challenge you will import several parquet files from Blob Storage into your Datawarehouse and then you will move data from staging area to the production one. 
+
+Here the proposed DB [database diagram](./Solutions/DedicatedSqlPool-TablesRelationships.pdf?raw=true).
```

Comment:
You have set this as a raw link here, but when you repeat it further down, you do not have it as a raw link.

Since this is the coach's guide, it is not required to be a raw link, but you should make it consistent throughout (whether it's raw or not raw).